blueprint/reviews/review-likes.mson

```diff
@@ -0,0 +1,15 @@
+
+## reviewLikes (object)
++ id: 1 (required)
++ type: reviewLikes (required)
++ links (object)
```

Comment:
@JasonLighthunter `id` and `type` would be the only required properties on `POST` and `PATCH` requests 👍